draft-ietf-quic-http.md

```diff
@@ -1071,7 +1071,8 @@ When a server receives this frame, it aborts sending the response for the
 identified server push.  If the server has not yet started to send the server
```

Comment:
How about reorganizing this section with one paragraph like this that starts with "When a server receives this frame, ..." and have another with "When a client receives this frame, ..." since the roles are completely asymmetric I believe?